source/index.js

```diff
@@ -40,6 +47,25 @@ module.exports = function(webDAVEndpoint, username, password) {
         // fs adapter type (for downstream integrations)
         [TYPE_KEY]: "webdav-fs",
 
+        /**
+         * Create a read stream for a remote file
+         * @param {String} filePath The remote path
+         * @param {CreateReadStreamOptions=} options Options for the stream
+         * @returns {Readable} A readable stream
+         */
+        createReadStream: function(filePath, options) {
+            var clientOptions = {};
+            if (options && options !== null) {
+                if (typeof options.headers === "object") {
+                    clientOptions.headers = options.headers;
+                }
+                if (typeof options.start === "number" && typeof options.end === "number") {
+                    clientOptions.range = { start: options.start, end: options.end };
+                }
```

Comment:
Hmm true. Don't recall where in the docs it said about the byte range start and end, but I guess we should mirror that. Unfortuantely [webdav-client has this requirement](https://github.com/perry-mitchell/webdav-client/blob/master/source/adapter/get.js#L78) also currently - It assumes both are there. Hmm..